### PR TITLE
Feature/sprint241023/#862kh0g2t customer service message menu

### PR DESCRIPTION
**Pull request by bulgarchik**

### CODE REVIEW

File: FastBank.Services/MessageService/IMessageService.cs [hunk 1]
```diff
@@ -24,6 +24,16 @@ public interface IMessageService
 
         public Message InputMessage(User user);
 
-        public void ShowMessagesMenu(User user);
+        public Message ReplayToMessage(User user, Message message);
```
Comment (AngelovGrowth) [inline FastBank.Services/MessageService/IMessageService.cs:27]: ReplayToMessage -> ReplyToMessage

File: FastBank.Services/MessageService/MessageService.cs [hunk 2]
```diff
@@ -75,19 +70,156 @@ public Message InputMessage(User user)
...
+                                        MessageType.Inquery, 
+                                        null);
+
+            _messageRepo.Add(replayMessage);
+            _messageRepo.UpdateStatus(message, MessageStatuses.Replied);
+
+            return replayMessage;
+        }
+
+        public void ShowMessagesMenu(User user, List<Message?>? messages = null)
```
Comment (AngelovGrowth) [inline FastBank.Services/MessageService/MessageService.cs:97]: Why do you expect to have List<Message?>?

Comment (bulgarchik) [reply FastBank.Services/MessageService/MessageService.cs:97]: image
On opening Customer Service menu I am getting messages to notify user about his new messages. So to reduce calls to db put it like argument to method. And that is why this method expect to have List<Message>

Comment (AngelovGrowth) [reply FastBank.Services/MessageService/MessageService.cs:97]: Ok, maybe it is good to review this

**Comment by AngelovGrowth** [comment]
@bulgarchik PR is ready to go

**Pull request merged**